rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/autoconfigure/RocketMQProperties.java

```diff
@@ -17,20 +17,20 @@
 
 package org.apache.rocketmq.spring.autoconfigure;
 
-import org.apache.rocketmq.common.MixAll;
-import org.springframework.boot.context.properties.ConfigurationProperties;
-
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import org.apache.rocketmq.common.MixAll;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @SuppressWarnings("WeakerAccess")
 @ConfigurationProperties(prefix = "rocketmq")
 public class RocketMQProperties {
 
     /**
-     * The name server for rocketMQ, formats: `host:port;host:port`.
+     * The name servers for rocketMQ, formats: `host:port,host:port`.
      */
-    private String nameServer;
+    private List<String> nameServer;
```

Comment:
The name seems a bit odd for a collections. Have you considered naming that `servers` ? If you want to retain backward compatibility, you could deprecate this one with a replacement metadata to `servers` and then apply the old logic if `nameServer` is set. If both are set, I would throw an exception.

You could then remove `nameServer` in the next feature release.